### DISCUSSION TITLE
Upgrading ignite project

### DISCUSSION THREAD

**Discussion by foratbahrani** [infinitered/ignite#2362] [Q&A]
Hi, I've made a project using ignite a few months ago, and want to update my project to latest version, your documentation file on upgrading ([here](https://github.com/infinitered/ignite/blob/master/docs/Upgrading-Ignite.md)) is very vague.

Can you help please?

**Comment by frankcalise** [comment]
Thanks for bringing that to our attention @foratbahrani. I think you're most likely referring to how do you make sure your already existing project is up to date since you saw a new release to Ignite?

In this case, react native, Expo and some packages were updated. This is at your own pace with your own project, you may or may not want to go to the later versions depending on your situation.

The same goes for the ignite directory in the project. You may have already customized this to your liking with your generators etc. We currently don't have some magical way to do a diff between the latest cli and the one your project was created with at the moment, so it is more eye-balling it.

Let me know if that helps, but I think that update doc is more for the globally installed command, which you don't need to really do if you use `npx ignite-cli...`
